transports/http/endpoints/api/tips/endpoints.go

```diff
@@ -63,24 +62,15 @@ func (h *handler) getTipLongestChain(c *gin.Context) {
 	c.JSON(http.StatusOK, newTipStateResponse(tip))
 }
 
-// PruneTip godoc.
-//
-//	@Summary Prune tip
-//	@Tags tip
-//	@Accept */*
-//	@Produce json
-//	@Success 200 {object} string
-//	@Router /chain/tip/prune/{hash} [get]
-//	@Param hash path string true "Requested Header Hash"
-//	@Security Bearer
-func (h *handler) pruneTip(c *gin.Context) {
-	param := c.Param("hash")
-	fmt.Println(param)
-	tip, err := h.service.GetPruneTip()
+// TO BE IMPLEMENTED
+// func (h *handler) pruneTip(c *gin.Context) {
```

Comment:
Maybe we could remove this completely and just add a task in JIRA to implement such endpoint? This code will not be very useful anyways :)

Reply:
removed